action.yml

```diff
@@ -24,9 +24,33 @@ inputs:
     required: false
     default: '/home/runner/work'
 
+
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: 'composite'
+  steps:
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      id: setup-python
+      with:
+        python-version: "3.11"
+        update-environment: false
+    - name: Setup venv
+      run: ${{ steps.setup-python.outputs.python-path }} -m venv ${GITHUB_ACTION_PATH}/venv
+
+      shell: bash -l {0}
+    - name: Install dependencies
+      run: ${GITHUB_ACTION_PATH}/venv/bin/python -m pip install -r ${GITHUB_ACTION_PATH}/requirements.txt
+      shell: bash -l {0}
+    - name: Run action
+      run: ${GITHUB_ACTION_PATH}/action_launcher.bash
+      shell: bash -l {0}
+      env:
+        INPUT_GITHUB_TOKEN: ${{ inputs.github_token }}
+        INPUT_CLANG_TIDY_FIXES: ${{ inputs.clang_tidy_fixes }}
+        INPUT_PULL_REQUEST_ID: ${{ inputs.pull_request_id }}
+        INPUT_REQUEST_CHANGES: ${{ inputs.request_changes }}
+        INPUT_SUGGESTIONS_PER_COMMENT: ${{ inputs.suggestions_per_comment }}
+        INPUT_REPO_PATH_PREFIX: ${{ inputs.repo_path_prefix }}
```

Comment:
Aren't these done implicitly?

Reply:
Not with the composite action setup. 
See [this run](https://github.com/platisd/clang-tidy-pr-comments/actions/runs/7532032616/job/20501857199#step:7:119) before I noticed that (ref https://github.com/platisd/clang-tidy-pr-comments/pull/62/commits/c56fa2aa68c16d32316dcf021676541e9ac0ffe0)